processor/attributesprocessor/testdata/config.yaml

```diff
@@ -342,6 +342,15 @@ processors:
       - key: token
         action: delete
 
+  attributes/from_context:
+    actions:
+    - key: origin
+      from_context: metadata.origin
+      action: insert
+    - key: enduser.id
+      from_context: auth.subject
```

Comment:
Can you refer to a specific server authenticator that produces `subject`?

Reply:
I hope this is useful enough. Probably needs help with wording.